specs/latest/1.0/index.html

```diff
@@ -753,6 +759,7 @@ <h3><a name="WEBGLCONTEXTATTRIBUTES">WebGLContextAttributes</a></h3>
     WebGLPowerPreference powerPreference = "default";
     boolean failIfMajorPerformanceCaveat = false;
     boolean desynchronized = false;
+    WebGLPixelFormat pixelFormat = "implicit";
```

Comment:
optional suggestion: `"implicit"` could instead be spelled as `undefined`/unspecified, by removing the default:
```suggestion
    WebGLPixelFormat pixelFormat;
```

Reply:
I can do that, though I need to change some surrounding text.